src/Grpc/Interop/test/testassets/InteropClient/InteropClient.csproj

```diff
@@ -4,8 +4,6 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <Nullable>enable</Nullable>
-    <!-- Suppress warnings about lowercase variable names in generated code -->
```

Comment:
This feels like a regression. Why are the analyzers complaining about generated code at all❔

Reply:
https://github.com/aspnet/AspNetCore-ManualTests/issues/1357